server/api/index.js

```diff
@@ -6,13 +6,42 @@
  * it can be accessed via /:resource
  */
 import {Router as router} from 'express';
+import {BigQuery} from '@google-cloud/bigquery';
+import mixedApi from './mixed-content';
 
 // TODO(ernestognw): Setup routes for every model in db
 const api = router();
+const bigqueryClient = new BigQuery();
 
 // Health check
 api.get('/', (req, res) => {
   res.status(200).send('API working');
 });
 
+// Health check (BigQuery)
+api.get('/test', async (req, res) => {
+  // The SQL query to run
+  const sqlQuery = `SELECT COUNT(*) requests,
+  COUNT(DISTINCT pageid) pages
+FROM httparchive.summary_requests.2018_09_01_desktop
+WHERE url NOT LIKE ("https%");`;
+
+  const options = {
+    query: sqlQuery,
+    location: 'US',
+  };
+
+  const [rows] = await bigqueryClient.query(options);
+
+  res.json({
+    description: 'Stackoverflow dataset query for testing the routes',
+    result: rows,
+  });
+});
+
+api.use('/mixed-content', (req, res, next) => {
```

Comment:
Is the `Access-Control-Allow-Origin` needed?

Maybe we can rewrite to only

```
api.use('/mixed-content', mixedApi);
```

But if it is needed, is likely that we will be repeating this header across endpoints. In that case, we can set the header for every request in app.js, where we do the `express.json()` part. We dan discuss this further.

Reply:
I got an error when I tried to make another endpoint without Access-Control-Allow-Origin, adding this header solved it

Reply:
Can I see the error? 

I think it is not needed since we're requesting from localhost, which is the same origin. Anyway, we can move it to app.js to get rid of this problem for every endpoint we have, but I'd prefer not to do it since it could be a security issue.